src/views/pages/Identity/index.jsx

```diff
@@ -14,12 +50,58 @@ function Identity() {
       </div>
       <div className="basis-3/4 px-8">
         <PageTitle>Identity</PageTitle>
-        {account ? (
-          <IdentityControls />
```

Comment:
I think we should keep the IdentityControls component as a sub-component to this page. It will help reduce the complexity.